packages/databases-collections/src/components/collection-fields/fle2-fields.jsx

```diff
@@ -16,8 +16,7 @@ import {
 import CollapsibleFieldSet from '../collapsible-field-set/collapsible-field-set';
 import FieldSet from '../field-set/field-set';
 
-// TODO(COMPASS-5777): Use URL specifically for FLE2
-const HELP_URL_FLE2 = 'https://www.mongodb.com/docs/drivers/security/client-side-field-level-encryption-guide/';
+const HELP_URL_FLE2 = 'https://dochub.mongodb.org/core/rqe-encrypted-fields';
```

Comment:
Should this also have the todo link update?
```suggestion
// TODO(COMPASS-5925): Use generic In-Use Encryption URL 
const HELP_URL_FLE2 = 'https://dochub.mongodb.org/core/rqe-encrypted-fields';
```

Reply:
No, because this one is specifically about Queryable Encryption, not the connection setup (which is shared between CSFLE & QE) :slightly_smiling_face: